superbench/benchmarks/micro_benchmarks/__init__.py

```diff
@@ -24,5 +25,5 @@
     'MicroBenchmark', 'MicroBenchmarkWithInvoke', 'ShardingMatmul', 'ComputationCommunicationOverlap', 'KernelLaunch',
     'CublasBenchmark', 'CudnnBenchmark', 'GemmFlopsBenchmark', 'CudaGemmFlopsBenchmark', 'MemBwBenchmark',
     'CudaMemBwBenchmark', 'DiskBenchmark', 'IBLoopbackBenchmark', 'CudaNcclBwBenchmark', 'RocmMemBwBenchmark',
-    'RocmGemmFlopsBenchmark', 'GpuSmCopyBwBenchmark'
+    'RocmGemmFlopsBenchmark', 'GpuSmCopyBwBenchmark','MemBwLatencyBenchmark'
```

Comment:
shall we change it to `CPUMemBwLatencyBenchmark`? because `MemBwBenchmark` is for GPU. Maybe there are some misunderstanding.

Reply:
good point. We can also change it to `HostMemBwLatencyBenchmark` if it makes more sense

Reply:
Thanks!